grpc/dataplane_grpc.proto

```diff
@@ -0,0 +1,75 @@
+syntax = "proto3";
+
+package config;
+
+option go_package = "dataplane/grpc"
+
+service ConfigService {
+    rpc GetGlobalConfig (GetGlobalConfigRequest) returns (GlobalConfig);
+    rpc UpdateGlobalConfig (UpdateGlobalConfigRequest) returns (UpdateGlobalConfigResponse);
+    rpc CheckConfigGeneration (CheckConfigGenerationRequest) returns (CheckConfigGenerationResponse);
+}
+
+message GetGlobalConfigRequest {}
+
+message UpdateGlobalConfigRequest {
+    GlobalConfig config = 1;
+}
+
+message UpdateGlobalConfigResponse {
+    bool status = 1;
+    string message = 2;
+}
+
+message CheckConfigGenerationRequest {}
+
+message CheckConfigGenerationResponse {
+    uint64 generation = 1;
+}
+
+message Subnet {
+    string name = 1;
+    string prefix = 2;
+    uint32 vni = 3;
+}
+
+message VRF {
```

Comment:
Should it be just named VPC? We're still thinking about it in terms of VPCs and VRF is just an implementation detail. Additionally, VPC name != VRF name and for logs/etc it would be better to use a VPC name while keeping VRF name separate.

Few more fields needed:

- VPC level VNI
- VRF name
- VPC ID (to be able to distinguish between VPCs re-created with the same name)